startup.d/20-syntax-check.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+python -m py_compile $(find . -name '*.py');
+if [ "$?" != "0" ]
+then
+  echo "There is syntax error. This may endanger chaos. Pauses updates."
+  killall bash
```

Comment:
Chaos does spawn a child for the web server, but unless something's happening in schedule, there's only one process managing PRs. That's also the original process started by Supervisor and the one overwritten by `startup.sh`.

If I'm missing a place where it forks, please point me to it.